src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -405,6 +466,14 @@ ${loaderJs}
 	initialize(content: string) {
 		this.webview = this._createInset(this.webviewService, content);
 		this.webview.mountTo(this.element);
+		this.webview.onDidFocus(() => {
+			if (this.activeCellId) {
+				this.webview.sendMessage({
```

Comment:
I'm wondering if this will break users' regular operations like clicking a button, selecting some text, etc.

Reply:
I added this.activeCellId = undefined after calling the focus listener. This should ensure that the message is sent only once per Ctrl + UpArrow in and will never send on focus via mouse. I'll leave this unresolved though in case I missed something.

Reply:
`activeCellId` will only be set after we focused the webview, then we set it to undefined here. I think this is fine for now, I will probably mess around with it. We saw a weird issue if we send this message immediately after calling `webview.focus()`

Reply:
I think this is fine for now, I will probably mess around with it. @rebornix We saw a weird issue if we send this message immediately after calling webview.focus() without waiting